src/content/guides/creating-clusters-via-crs-on-azure/index.md

```diff
@@ -189,3 +189,27 @@ There are also specific reference pages for [cluster templating](/reference/kube
 As a result of rendering the CRs ([sample](/reference/kubectl-gs/template-cluster/#example)), a user will get YAML manifests containing valid CRs that can create a tenant cluster and its node pools.
 The resources can then be created by applying the manifest files to the Control Plane, e.g. `kubectl create -f <cluster manifest file>.yaml`.
 Of course, that requires the user to be authorized towards Kubernetes Control Plane API.
+
+## How to configure OIDC authentication at cluster creation using Cluster API
+
+Starting from version {{% first_azure_nodepools_version %}} on Azure we have enabled the possibility to configure the OIDC per cluster at the cluster creation stage. This feature only includes configuration of the OIDC in the `Cluster` CR before it is applied on the Control Plane.
+
+In order to configure your new cluster with OIDC, you will have to add annotations to the Cluster CR as following:
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1alpha3
+kind: Cluster
+metadata:
+  annotations:  
+    oidc.giantswarm.io/client-id: OIDC_CLIENT_ID
+    oidc.giantswarm.io/group-claim: GROUP_CLAIM
+    oidc.giantswarm.io/issuer-url: https://login.microsoftonline.com/TENANT_ID/v2.0
+    oidc.giantswarm.io/username-claim: USERNAME_CLAIM
+```
+
+This will result in setting up the OIDC config per cluster in the K8S API manifest.
+Currently changing/adding config to already existing cluster is not fully supported at Giant Swarm. Please talk to your SE if there is any need to change those settings.
+
+## How to delete cluster via Control Plane API
+
+In order to delete your cluster created via the Cluster API on the Control Plane, simply delete the `Cluster` CR corresponding to the given cluster. This will result in starting the process of cluster deletion and clean up of all cluster CRs on the Control Plane.
```

Comment:
```suggestion
In order to delete your cluster created via the Cluster API on the Control Plane, simply delete the `Cluster` CR corresponding to the given cluster. This will result in starting the process of cluster deletion and clean up of all related CRs on the Control Plane.
```